### PR TITLE
ARC-2033 add col to record success time

### PR DESCRIPTION
**Pull request by gxueatlassian**
**What's in this PR?**
Add db cols for recording success time for each entities. 

**Why**
Even though it is not being used right now, we don't want to lose the information.

**Added feature flags**
N/A

**Affected issues**  
ARC-2033

**How has this been tested?**  
N/A

**Whats Next?**
Impl code to save the time.

### CODE REVIEW

**Review by joshkay10 — APPROVED** [review]

File: db/migrations/20230320085600-add-commit-from-all-entities.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.addColumn("RepoSyncStates", "branchFrom", { type: Sequelize.DATE, allowNull: true });
+    await queryInterface.addColumn("RepoSyncStates", "pullFrom", { type: Sequelize.DATE, allowNull: true });
+    await queryInterface.addColumn("RepoSyncStates", "buildFrom", { type: Sequelize.DATE, allowNull: true });
+    await queryInterface.addColumn("RepoSyncStates", "deploymentFrom", { type: Sequelize.DATE, allowNull: true });
```
Comment (Harminder84) [inline db/migrations/20230320085600-add-commit-from-all-entities.js:8]: What about `issueFrom`? is it skipped intentionally?

Comment (gxueatlassian) [reply db/migrations/20230320085600-add-commit-from-all-entities.js:8]: I looked at that, didn't think we have an task for issue? Although we have cols.

Comment (Harminder84) [reply db/migrations/20230320085600-add-commit-from-all-entities.js:8]: We can remove the `issue` column if it is obsolete one; may be in a seperate ticket.

Comment (gxueatlassian) [reply db/migrations/20230320085600-add-commit-from-all-entities.js:8]: Yup. Definitely in another PR.

**Pull request merged**